app/spec/instance-event/commons.spec.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+describe("Commons for instance events", function() {
+  const fs = require("fs");
+  const path = require("path");
+
+  const config = require(path.resolve("src/utils/config"));
+
+  const cmns = require(path.resolve(
+    "src/" + config.getJobNameForInstanceEvent() + "/commons"
+  ));
+
+  it("should substitute aliases present in an instance definition.", function() {
```

Comment:
@mks-d , note that I am not really testing extensively this function since if I understand correctly, the instance passed to the function has already being verified. And this function wouldn't be used (at least as it is) outside of this job's scope.
I am only verifying that the *uuid* is indeed correctly substituted.